contracts/LoanManager.sol

```diff
@@ -39,14 +39,15 @@ contract LoanManager is Restricted, TokenReceiver {
         uint32 productId; // 3
         LoanState state; // 4
         uint40 maturity; // 5
+        uint marginRate; // 6
```

Comment:
we don't need to store it in Loan because loanProducts are immutable (only enable/disable)

Reply:
+ `getLoanTuple` needs to be expanded (maybe we won't need to keep the old getter for compatibility as we just add one more item to the end of the tuple?)

Reply:
It was not intended to be the final name, as this feature is still uder development. "marginRate" simply means the "rate" where the "margin" is. 
I'm not against a better name, but "marginCallExchangeRate" feels much more confusing to me. 

Yes, getLoanTuple has to be extended to. There is still a lot of stuff to do.
No tests or Sb scripts use getLoanTuple at all, that is only for the frontend.

Reply:
We have settled on "marginCallRate" with Charlie.